internal/api/keppel/accounts.go

```diff
@@ -62,11 +62,11 @@ type Account struct {
 
 // ReplicationPolicy represents a replication policy in the API.
 type ReplicationPolicy struct {
-	Strategy string
+	Strategy string `json:"strategy"`
 	//only for `on_first_use`
-	UpstreamPeerHostName string
+	UpstreamPeerHostName string `json:"upstream_peer_hostname"`
 	//only for `from_external_on_first_use`
-	ExternalPeer ReplicationExternalPeerSpec
+	ExternalPeer ReplicationExternalPeerSpec `json:"external_peer"`
 }
```

Comment:
For a PoC, this is fine. For the final implementation, this should move into `package keppel`. In general, to keep the dependency graph orderly, my rule of thumb is to put everything that drivers need into `package keppel`, and everything that APIs and janitor tasks need into one of `package keppel/processor/auth`. There should not be any horizontal dependencies between APIs, janitor tasks and drivers.

The existing Policy types in `package keppel` can serve as a guideline to inform the kind of interface offered to the API and driver implementation.